.github/workflows/core_build_publish.yml

```diff
@@ -90,6 +90,7 @@ jobs:
       - name: Tool Restore
         run: dotnet tool restore
       - name: Check formatting
+        if: false
```

Comment:
Why?

Reply:
The version used by the project does not work with .NET 9, thus I disabled it temporarily (fixed in next PR).